### DISCUSSION TITLE
oc 4.11.0-0.okd-2022-08-20-022919 fails cli login

### DISCUSSION THREAD

**Discussion by bdlink** [okd-project/okd#1332] [Releases]
The current oc (on the Intel Mac at least, from [openshift-client-mac-4.11.0-0.okd-2022-08-20-022919.tar.gz](https://github.com/okd-project/okd/releases/download/4.11.0-0.okd-2022-08-20-022919/openshift-client-mac-4.11.0-0.okd-2022-08-20-022919.tar.gz)) fails to login from a terminal with message 
error: x509: “kube-apiserver-lb-signer” certificate is not trusted

This is the same using a token (from the console) or a username.

The server is at version 4.11.0-0.okd-2022-08-20-022919.  
Once logged in, the other oc commands seem to work fine.

Previous version [openshift-client-mac-4.11.0-0.okd-2022-07-29-154152.tar.gz](https://github.com/okd-project/okd/releases/download/4.11.0-0.okd-2022-07-29-154152/openshift-client-mac-4.11.0-0.okd-2022-07-29-154152.tar.gz) has same error.

versions from [openshift-client-mac-4.10.0-0.okd-2022-07-09-073606.tar.gz](https://github.com/okd-project/okd/releases/download/4.10.0-0.okd-2022-07-09-073606/openshift-client-mac-4.10.0-0.okd-2022-07-09-073606.tar.gz) work (that is what I was using until yesterday).

I will check windows and Linux:
oc 4.11.0-0.okd-2022-08-20-022919 works on WIndows 10
oc 4.11.0-0.okd-2022-08-20-022919 works on Fedora 36

**Comment by fmerici** [comment]
Any update? I just updated to `oc 4.11.6` and issue still present
